src/types/PersonSex.ts

```diff
@@ -0,0 +1,4 @@
+export enum PersonSex {
+  female = 'f',
+  male = 'm',
```

Comment:
```suggestion
  Female = 'f',
  Male = 'm',
```

Also would be good to add `All` option